### ISSUE TITLE
[Bug]: Document 'relations' editable adds only first selected object

### ISSUE THREAD

**Issue by thijs-windels** [pimcore/pimcore#12604]
### Expected behavior

When selecting multiple objects in a relations editable, I'd expect they would all be loaded.
![image](https://user-images.githubusercontent.com/23450920/177555602-82238aa9-419e-46ba-bc9a-593f2f858f50.png)
![image](https://user-images.githubusercontent.com/23450920/177555940-2234d5f7-d92d-4fbb-9137-b3e96a84ae3b.png)

### Actual behavior

As shown in the screenshots, only the first one is added.

Looks like you the component gets reloaded too soon in the for loop (see `pimcoreadmin/js/pimcore/document/editables/relations.js`). At least, that worked for me.

```php
    addDataFromSelector: function (items) {
        if (items.length > 0) {
            for (var i = 0; i < items.length; i++) {
                if (!this.elementAlreadyExists(items[i].id, items[i].type)) {

                    var subtype = items[i].subtype;
                    if (items[i].type == "object") {
                        if (items[i].subtype == "object") {
                            if (items[i].classname) {
                                subtype = items[i].classname;
                            }
                        }
                    }

                    this.store.add({
                        id: items[i].id,
                        path: items[i].fullpath,
                        type: items[i].type,
                        subtype: subtype
                    });

                    ====================================================
                      This should be outside of the for loop
                    ====================================================
                    if (this.config.reload) {
                        this.reloadDocument();
                    }
                    ====================================================
                }
            }
        }
    },
```

### Steps to reproduce

- Add a relations editable to your document
- Click 'select'
- Only first element is added

**Comment by brusch** [comment]
@DeChaun Thanks for reporting the issue. May I ask you to create a pull request as you have already a working solution? Thanks in advance 😊

**Comment by dvesh3** [comment]
Fixed by #12613
